web_src/css/repository.css

```diff
@@ -591,85 +591,83 @@
-  margin-bottom: 0.5rem;
+.issue-title-header {
+  width: 100%;
+  padding-bottom: 4px;
+  margin-bottom: 1rem;
 }
 
-.repository.view.issue .title .issue-title.edit-active {
+.issue-title-meta {
   display: flex;
   align-items: center;
 }
 
-.repository.view.issue .title .issue-title.edit-active h1 {
+.repository.view.issue .issue-title-buttons,
+.repository.view.issue .edit-buttons {
   display: flex;
-  width: 100%;
 }
 
 @media (max-width: 767px) {
-  .repository.view.issue .title .issue-title.edit-active {
+  .repository.view.issue .issue-title {
     flex-direction: column;
   }
-  .repository.view.issue .title .issue-title.edit-active h1 {
-    margin-right: 0;
-    margin-bottom: 1rem;
-    padding-right: 0;
-  }
-  .repository.view.issue .title .issue-title.edit-active h1 .ui.input input {
-    width: calc(100% - 2rem);
-  }
-  .repository.view.issue .title .issue-title.edit-active .edit-buttons {
-    padding-bottom: 1rem;
-    width: 100%;
-  }
-  .repository.view.issue .title .issue-title.edit-active .edit-buttons .button {
-    width: 100%;
-    margin-right: 0.5rem;
-  }
-  .repository.view.issue .title .issue-title.edit-active .edit-buttons .button:last-child {
-    margin-right: 0;
+  .repository.view.issue .edit-buttons {
+    margin-top: .5rem;
   }
 }
 
-.repository.view.issue .title .issue-title h1 {
+.repository.view.issue .issue-title {
+  display: flex;
+  align-items: center;
+  margin-bottom: 8px;
+}
+
+.repository.view.issue .issue-title h1 {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  width: 100%;
   font-weight: 300;
-  font-size: 2.3rem;
+  font-size: 32px;
+  line-height: 40px;
   margin: 0;
-  padding-right: 0.5rem;
+  padding-right: 0.25rem;
+  min-height: 41px; /* avoid layout shift on edit */
 }
 
-.repository.view.issue .title .issue-title h1 .ui.input {
+.repository.view.issue .issue-title h1 .ui.input {
   font-size: 0.5em;
-  width: 100%;
 }
 
-.repository.view.issue .title .issue-title h1 .ui.input input {
+.repository.view.issue .issue-title h1 .ui.input input {
   font-size: 1.5em;
-  padding: 6px 1rem;
+  padding: 2px .5rem;
 }
 
-.repository.view.issue .title .issue-title .edit-button {
-  float: right;
-  padding-left: 1rem;
+.repository.view.issue .issue-title .index {
+  color: var(--color-text-light-2);
 }
 
-.repository.view.issue .title .issue-title .edit-buttons {
-  display: flex;
+.repository.view.issue .issue-title .label {
+  margin-right: 10px;
 }
 
-.repository.view.issue .title .issue-title .index {
-  color: var(--color-text-light-2);
+.repository.view.issue .issue-title .edit-zone {
+  margin-top: 10px;
 }
 
-.repository.view.issue .title .issue-title .label {
-  margin-right: 10px;
+.issue-state-label {
+  font-size: 14px !important;
+  padding: 5px 10px !important;
+  border-radius : 6px !important;
+  margin-left: 0 !important;
+  margin-right: 8px !important;
+  display: inline-flex !important;
+  vertical-align: middle !important;
```

Comment:
I still have the question what's the difference between this CSS class and the existing `ui xxx label`?

Is the "original one" good enough?

If it is for general purpose, could it be our framework-level CSS class? I really hope Gitea could have some uniform UI components, instead of patching them one by one.

Reply:
I am primarly trying to closely replicate GitHub's label and Fomantic label classes just are not suitable for it.

Reply:
Removed a few styles on it, now down to 5 rules.

Reply:
Let's try it